cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ModelUtils.java

```diff
@@ -434,4 +435,57 @@ public static String getJavaSystemPropertiesToString(List<SystemProperty> javaSy
         }
         return String.join(" ", javaSystemPropertiesList);
     }
+
+    /**
+     * This method transforms a String into a List of Strings, where entry = line from input.
+     * The lines beginning with '#' (comments) are ignored.
+     * @param config ConfigMap data as a String
+     * @return List of String key=value
+     */
+    public static List<String> getLinesWithoutCommentsAndEmptyLines(String config) {
+        List<String> validLines = new ArrayList<>();
+        if (config != null) {
+            List<String> allLines = Arrays.asList(config.split("\\r?\\n"));
+
+            for (String line : allLines) {
+                if (!line.startsWith("#") && !line.isEmpty()) {
+                    validLines.add(line);
+                }
+            }
+        }
+        return validLines;
+    }
+
+    /**
+     * Gets data from ConfigMap as a String
+     * @param configMap ConfigMap to get data from
+     * @param key key to get data from ConfigMap
+     * @return String value of ConfigMap data entry. If the key does not exist in the ConfigMap, return null
+     */
+    public static String getDataFromConfigMap(ConfigMap configMap, String key) {
```

Comment:
I still don't understand why do you need to go configuration -> config map -> configuration instead of just using the configuration.